packages/expo/package.json

```diff
@@ -47,7 +47,8 @@
     "typescript": "^3.3.3"
   },
   "dependencies": {
-    "@bugsnag/core": "^5.2.0",
+    "@bugsnag/core": "^6.0.0",
+    "@bugsnag/plugin-react-native-global-error-handler": "^5.2.0",
```

Comment:
What's up with the versioning differing from core?

Reply:
Hmm, just a hangover from starting this branch before 6.0.0 went out I think. It doesn't really matter because it's just linking stuff up locally and upon publish it'll get a proper version, but I'll sync it up now for tidiness.